src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonItem } from '../PersonItem';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  return (
+    <div className="box table-container">
+      <table
+        data-cy="peopleTable"
+        className="table is-striped is-hoverable
+      is-narrow is-fullwidth"
```

Comment:
The className string is broken into two lines. It should be properly formatted within one line.